### ISSUE TITLE
Can't scroll multiple characters/lines when holding down hjkl

### ISSUE THREAD

**Issue by rolodato** [VSCodeVim/Vim#422]
### Issue Summary

Using `hjkl` to move the cursor in normal mode only moves one character while held down.

### Steps to Reproduce

1. Open any file
2. In normal mode, try to scroll by holding down `h`, `j`, `k` or `l`
3. Cursor only advances one line/character

This is not how regular vim works, so seems like a bug. Scrolling with the arrow keys in VSCode works as expected too.

### Technical details:

* VSCode Version: 1.3.0
* VsCodeVim Version: 0.1.1
* OS: OS X 10.11.5

**Comment by rebornix** [comment]
@rolodato people ran into this issue before and we found out it's likely an OSX or VS Code issue, see here https://github.com/VSCodeVim/Vim/issues/345 .

Can you help try the commands we mentioned in #345 and let me know if that works? Thanks in advance.

If it works, now I need to figure out whether it's due to VS Code.

**Comment by rolodato** [comment]
This worked after restarting VS Code:

```sh
defaults write -g ApplePressAndHoldEnabled -bool false
```

Oddly enough, this did not:

```sh
defaults write -g ApplePressAndHoldEnabled -bool true
defaults write com.microsoft.VSCode ApplePressAndHoldEnabled -bool false
```

Changing this globally works, but it would be nice to have this change only scoped to VS Code. Maybe the editor window runs on a separate domain? I don't know how I could find that out.

**Comment by rebornix** [comment]
@rolodato  that also bothers me why we can only changw the default setting to make it work. Seems we are not doing the same thing as Sublime. I'll come back to you if I have more info about this and put it in our doc.

**Comment by johnfn** [comment]
@rebornix beat me to the punch again. 

Gonna close this; hopefully it can be mitigated upstream.

**Comment by siddharth-krishna** [comment]
```defaults write com.microsoft.VSCode ApplePressAndHoldEnabled -bool false
```
didn't work for me either, I had to use
```defaults write -g ApplePressAndHoldEnabled -bool false
```
to get it to work. Maybe we could change what the extension page says until we figure out how to get the non-global setting to work?

**Comment by jpoon** [comment]
@siddharth-krishna what do you mean? What do you suggest we change it to?

**Comment by siddharth-krishna** [comment]
Well, the consensus on both this thread as well as issue #345 seems to be that writing the default globally is the only way to get repeated keypresses to register. But the extension home page (at least the one I saw when I installed the extension) still says write the defaults for `com.microsoft.VSCode`, which didn't work for me. Maybe the home page could say, one way to get it to work is to set the default globally, but be careful as it changes global behaviour? Or did I misunderstand the discussion on this issue?
